src/components/products/guides/index.js

```diff
@@ -116,6 +116,17 @@ export const guides = {
             },
           ],
         },
+        {
+          title: 'Templates',
+          links: [
+            {
+              title: 'NextJs Template',
+              href: '/guides/templates/nextjs',
```

Comment:
seo wise: should we add solana, umi or something like that into the uri?

Reply:
yeah I came up with a naming convention for templates a number of weeks ago

`metaplex-reactframework-componentLib-cssframework-template`

Examples
`metaplex-nextjs-tailwind-template`
`metaplex-nextjs-shadcn-tailwind-template`
`metaplex-nextjs-manetime-template`
`metaplex-react-manetime-template`
`metaplex-react-tailwind-template`

I should probably make the URI and title based on this.